src/typescript/frontend/src/middleware.ts

```diff
@@ -64,5 +64,5 @@ export default async function middleware(request: NextRequest) {
 }
 
 export const config = {
-  matcher: "/((?!verify|api|_next/static|_next/image|favicon.ico|logo192.png|manifest.json).*)",
+  matcher: "/((?!verify|api|_next/static|_next/image|favicon.ico|logo192.png|okx-logo.png|manifest.json).*)",
```

Comment:
Just read the docs on this. Do you mind also adding items from this:

```
if (
    pathname === "/social-preview.png" ||
    pathname === "/webclip.png" ||
    pathname === "/icon.png" ||
    pathname === "/test" ||
    pathname === "/geolocation" ||
    pathname === "/verify_status"
  ) {
    return NextResponse.next();
}
```

I think that `*.png` could be added. Also, is `geolocation` needed anymore ?

Reply:
Tried to resolve this in: a92c191433dd508a33c873a448092c95b255727b

Will need to check manually (I'll check on preview build that middleware isn't running on /test)